modules/ROOT/pages/metrics-archive-api.adoc

```diff
@@ -0,0 +1,33 @@
+= Anypoint Monitoring Archive API
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+The Anypoint Monitoring Archive API enables you to download metric data for APIs. The data that this API archives appears in your xref:monitoring::api-analytics-dashboard.adoc#enhanced-api-metrics[enhanced API metrics dashboards]. However, the data contained in the archive is asynchronous.
+
+You must have a Titanium subscription to use this API.
```

Comment:
There is a rate limit on using this API. The rate limit is 18,000 calls per hour.